Orange/widgets/unsupervised/owtsne.py

```diff
@@ -513,7 +513,7 @@ def run(self):
 
     def __ensure_task_same_for_pca(self, task: Task):
         assert self.data is not None
-        assert task.data is self.data
+        np.testing.assert_array_equal(task.data.X, self.data.X)
```

Comment:
`numpy.testing` should not be used outside of unittests.

I am assuming the cause of this assert violation is a [check](https://github.com/biolab/orange3/blob/06f7630d8753143f4c7f6b029c7dc79264d76318/Orange/widgets/visualize/utils/widget.py#L453) that the data is the same upon receiving the input but only after having already updated `self.data`. 

If that is the case I would just remove the assert.

Reply:
It's removed.